examples/multimodal_video/utils/protocol.py

```
@@ -0,0 +1,169 @@
+
+
+class TextContent(BaseModel):
+    type: Literal["text"]
+    text: str
+
+
+class ImageURLDetail(BaseModel):
+    url: str
+
+
+class ImageContent(BaseModel):
```

Pylint (3.3.7)

[refactor] Too few public methods (0/2)

(R0903)